qdrant_docs_mcp/tools/importer.py

```diff
@@ -152,14 +153,28 @@ def clone_repo(url: str) -> Generator[Path]:
 
 
 def get_library_config(library: Library, repo: Path) -> LibraryConfig:
-    return LibraryConfig.model_validate_json((repo / library.config_file).read_text())
+    if isinstance(library.config_file, LibraryConfig):
+        return library.config_file
+
+    if (repo / library.config_file).is_file():
```

Comment:
shouldn't we emit a warning / raise an exception / whatever here?

`config_file` can either be a path or LibraryConfig instance, what is it, if it's neither of these two?

Reply:
What is the default value of `config_file`?

I mean what if a path to a config file was written wrong? We'll just silently use the default config?